core/node/rpc/debug.go

```diff
@@ -63,6 +63,7 @@ func (s *Service) registerDebugHandlers(enableDebugEndpoints bool, cfg config.De
 	mux := s.mux
 	handler := debugHandler{}
 	mux.HandleFunc("/debug", handler.ServeHTTP)
+	handler.HandleFunc(mux, "/debug/storage", s.handleDebugStorage)
```

Comment:
Also please add an option to enable/disable it here (and set it to true in GetDefaultConfig())

https://github.com/river-build/river/blob/main/core/config/config.go#L377